prettier.config.js

```diff
@@ -1,11 +0,0 @@
-// @flow
-
-module.exports = {
```

Comment:
Why? Isn't it better to have all our configuration in js?

Reply:
Easier to use yml, it's text only. Shorter file name. Dot file name, more hidden in the src tree.

Reply:
My order of preference for configuration files is: `js > yml > json`. But as you prefer :).